frontend/packages/console-shared/src/components/popover/SizeAwarePopover.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import Measure from 'react-measure';
+import { Popover, Button } from '@patternfly/react-core';
+import './SizeAwarePopover.scss';
+
+type SizeAwarePopoverProps = {
+  content: string;
+};
+
+const SizeAwarePopover: React.FC<SizeAwarePopoverProps> = ({ content }) => {
+  const [isContentClamped, setIsContentClamped] = React.useState<boolean>(false);
+  const onResize = (offset, scroll) => {
+    if (scroll > offset) {
+      setIsContentClamped(true);
+    }
+  };
+  return (
+    <Measure
+      bounds
+      offset
+      scroll
+      onResize={(contentRect) => {
```

Comment:
Use only a single inline function that accepts contentRect and remove onResize.
No need for 2 functions.